packages/central-server/src/tests/apiV2/helpers.test.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import { ValidationError } from '@tupaia/utils';
+import { getQueryOptionsForColumns } from '../../apiV2/GETHandler/helpers';
+
+const baseRecordType1 = 'survey_response';
+const baseRecordType2 = 'project';
+const columnNames1 = ['survey_response.id', 'survey.name'];
+const columnNames2 = ['survey_response.id', 'survey.name', 'user_account.first_name'];
+const columnNames3 = ['survey_response.id', 'country.name'];
+const columnNames4 = ['survey_response.id', 'country.name', 'disaster.type'];
+const columnNames5 = ['_survey_response.id', 'survey.name'];
```

Comment:
```suggestion
const columnNames5 = ['survey_response.id', 'survey.name'];
```

Reply:
Ah I see, this is testing a specific case. Because the data here is not huge, I would actually recommend inlining it so that within each test you can see what is going on and why